russian/kompot/kompot.md

```diff
@@ -0,0 +1,4 @@
+Компот забродил и вы тоже. 
+
+1. [И тут вы просыпаетесь](wakeup/wakeup.md)
+2. [Вы продолжаете бродить](walk/walk.md)
```

Comment:
There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.